README.md

```diff
@@ -18,12 +18,12 @@ It relies on a few libraries:
 * [Boost](http://www.boost.org/) 1.40 or later.
   Required libraries: *thread*, *system*, *regex*, and *date_time*.
   Optional library for Boost Test driver: *test*.
-* [GTest](http://code.google.com/p/googletest/) 1.4 or later.
-  Optional for the GTest driver.
+* [GTest](http://code.google.com/p/googletest/) 1.6 or later.
```

Comment:
It's already there:


    It might work with earlier versions of the libraries, but it was not
    tested with them.